code/game/objects/items/npc_beacon.dm

```diff
@@ -0,0 +1,153 @@
+
+//the holdable item
+/obj/item/campaign_beacon/npc_beacon
+	name = "\improper N.P.C. beacon"
+	desc = "A bulky device that is used to teleport emergency support units directly into battle. Often used for garrisoning important locations."
+	icon = 'icons/obj/items/npc_beacon.dmi'
+	icon_state = "beacon_undeployed"
+	base_icon_state = "beacon"
+	deployable_type = /obj/structure/npc_beacon
+	deploy_time = 2 SECONDS
+	pixel_w = -4
+	w_class = WEIGHT_CLASS_BULKY
+
+/obj/item/campaign_beacon/npc_beacon/examine(mob/user)
+	. = ..()
+	. += span_notice("This one belongs to [faction ? faction : "no one"].")
+
+/obj/item/campaign_beacon/npc_beacon/tgmc
+	deployable_type = /obj/structure/npc_beacon/tgmc_standard
+	faction = FACTION_TERRAGOV
+
+/obj/item/campaign_beacon/npc_beacon/tgmc/big
+	icon_state = "fc_beacon_undeployed"
+	base_icon_state = "fc_beacon"
+	deployable_type = /obj/structure/npc_beacon/tgmc_big
+
+/obj/item/campaign_beacon/npc_beacon/som
+	deployable_type = /obj/structure/npc_beacon/som_standard
+	faction = FACTION_SOM
+
+/obj/item/campaign_beacon/npc_beacon/som/big
+	icon_state = "fc_beacon_undeployed"
+	base_icon_state = "fc_beacon"
+	deployable_type = /obj/structure/npc_beacon/som_big
+
+//The deployed beacon, although this could be map spawned as well
+/obj/structure/npc_beacon
+	name = "\improper N.P.C. beacon"
+	desc = "A bulky device that is used to teleport emergency support units directly into battle. Often used for garrisoning important locations."
+	icon = 'icons/obj/items/npc_beacon.dmi'
+	icon_state = "beacon_activating"
+	base_icon_state = "beacon"
+	density = FALSE
+	resistance_flags = RESIST_ALL
+	pixel_w = -8
+	///List of jobs that are spawned by this item
+	var/list/job_list
+
+/obj/structure/npc_beacon/Initialize(mapload, obj/item/_internal_item, mob/deployer)
+	. = ..()
+	addtimer(CALLBACK(src, PROC_REF(spawn_mobs)), 3 SECONDS)
```

Comment:
so this only deploys on the first spawn? ALso would they spawn if we spawn the item instead of the object directly

Reply:
no, the structure cannot be undeployed, and no, the item doesn't spawn anything.